checkov/terraform/checks/resource/ncp/NKSControlPlaneLogging.py

```diff
@@ -0,0 +1,20 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceCheck
+
+
+class NKSControlPlaneLogging(BaseResourceCheck):
```

Comment:
would be great if you can change it to a `BaseResourceValueCheck`, makes it much easier 🙂

Reply:
thank your advice. I fixed my code.